pom.xml

```diff
@@ -56,4 +78,16 @@
             <email>janarioliver@gmail.com</email>
         </developer>
     </developers>
+
+    <build>
+        <pluginManagement>
```

Comment:
if it works, then ok. I thought that you need to set plugin directly in build tag